doc/source/user-guide/configuration.rst

```diff
@@ -0,0 +1,78 @@
+.. _ref_user_guide_configuration:
+
+Basic configuration
+===================
+
+Add the following to your `conf.py` file to use the Ansys Sphinx theme:
+
+.. code-block:: python
+
+   html_theme = "ansys_sphinx_theme"
+
+From ansys sphinx theme you can use the following features:
+
+1. PyAnsys and Ansys logos
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Ansys Sphinx theme includes the PyAnsys and Ansys logos. All The logos
+are available in the
+`ansys_sphinx_theme/static/ <static_files_>`_
+directory. You can use the following code to add the logos to your documentation:
+
+.. code-block:: python
+
+    from ansys_sphinx_theme import pyansys_logo_black, ansys_logo_black, ansys_favicon
+
+    html_logo = pyansys_logo_black
+    html_favicon = ansys_favicon
+
+``favicon`` is the icon that appears in the browser tab.
+
+#. **Version switcher**
```

Comment:
This is getting rendering inside of previous subsection rather than being a subsection on its own.

Reply:
added